src/games/strategy/util/UrlStreams.java

```diff
@@ -80,7 +78,6 @@ protected UrlStreams(Function<URL, URLConnection> connectionFactory) {
       connection.setUseCaches(false);
       return Optional.of(connection.getInputStream());
     } catch (IOException e) {
-      ClientLogger.logError("Failed to open a connection to: " + url, e);
```

Comment:
The answer is in the title: "Do not log errors when a map was deleted out from under the game."

The more interesting answer is that the game code determines that the map is missing by failing to read it. Implied in this is any client will need to log an error if they see an Optional.empty and if they cannot handle the situation.